platform/darwin/src/MGLNetworkConfiguration.mm

```diff
@@ -66,6 +67,19 @@ - (NSURLSessionConfiguration *)defaultSessionConfiguration {
     return sessionConfiguration;
 }
 
+- (void)setStopsRequests:(BOOL)stopsRequests {
+    if (stopsRequests) {
+        mbgl::NetworkStatus::Set(mbgl::NetworkStatus::Status::Offline);
```

Comment:
I could be wrong, but this will be Offline forever. Because once you set the status to `Offline`, the following code will always be true.

```
- (BOOL)stopsRequests {
    auto status = mbgl::NetworkStatus::Get();
    return status == mbgl::NetworkStatus::Status::Offline;
}
```